packages/entities/entities-plugins/src/components/PluginForm.cy.ts

```diff
@@ -0,0 +1,1273 @@
+        props: {
+          config,
+          pluginType: 'cors',
+          useCustomNamesForPlugin: true,
+        },
+        router,
+      })
+
+      cy.wait('@getPluginSchema')
+      cy.wait('@getScopedEntity')
+      cy.get('.kong-ui-entities-plugin-form-container').should('be.visible')
+
+      cy.get('.Scoped-check input').should('be.visible')
+      cy.get('.Scoped-check input').should('have.value', '1')
+      cy.get('.field-selectionGroup .field-AutoSuggest').should('be.visible')
+      cy.get('#service-id').should('be.visible')
+      cy.getTestId(`k-select-item-${scopedService.id}`).find('.selected').should('exist')
+    })
+
+    it('should pick correct url while creating plugin', () => {
+      interceptKonnectSchema()
+      interceptKonnectValidatePlugin()
+      interceptKonnectCreatePlugin()
+
+      cy.mount(PluginForm, {
+        global: { components: { VueFormGenerator } },
+        props: {
+          config: baseConfigKonnect,
+          pluginType: 'cors',
+          useCustomNamesForPlugin: true,
+        },
+        router,
+      })
+
+      cy.wait('@getPluginSchema')
+      cy.get('.kong-ui-entities-plugin-form-container').should('be.visible')
+
+      cy.get('#instance_name').type('kai_cors_plugin')
+      cy.get('#tags').type('tag1,tag2')
+
+      cy.getTestId('form-submit').click()
+      cy.wait('@validatePlugin')
+      cy.wait('@createPlugin')
+    })
+
+    it('should pick correct url while creating plugin credential', () => {
+      const config = { ...baseConfigKonnect, entityId: scopedConsumer.item.id, entityType: 'consumers' }
+      interceptKonnectCreatePlugin({ credential: true, entityId: scopedConsumer.item.id })
+
+      cy.mount(PluginForm, {
+        global: { components: { VueFormGenerator } },
+        props: {
+          config,
+          credential: true,
+          pluginType: 'acl',
+          hideScopeSelection: true,
+          useCustomNamesForPlugin: true,
+        },
+        router,
+      })
+
+      cy.get('.kong-ui-entities-plugin-form-container').should('be.visible')
+
+      cy.get('#group').type('kai_group')
+      cy.get('#tags').type('tag1,tag2')
+
+      cy.getTestId('form-submit').click()
+      cy.wait('@createPlugin')
+    })
+
+    it('should show edit form', () => {
+      const config = { ...baseConfigKonnect, entityId: scopedService.id, entityType: 'services' }
+      interceptKonnectSchema()
+      interceptKonnectOperatePlugin({ method: 'GET', alias: 'getPlugin', id: plugin1.id })
+      interceptKonnectScopedEntity({ entityType: config.entityType })
+
+      cy.mount(PluginForm, {
+        global: { components: { VueFormGenerator } },
+        props: {
+          config,
+          pluginType: 'cors',
+          pluginId: plugin1.id,
+          useCustomNamesForPlugin: true,
+        },
+        router,
+      })
+
+      cy.wait('@getPluginSchema')
+      cy.wait('@getPlugin')
+      cy.wait('@getScopedEntity')
```

Comment:
suggestion: you can use wait without strict order like 
`cy.wait(['@getPluginSchema', '@getPlugin', '@getScopedEntity']).then`